### PR TITLE
DEVX-4911 Retrieve records from list of UUIDs

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Reports API: Get records synch by ID. You can now specify a comma-separated list of UUIDs.

[DEVX-4911](https://nexmoinc.atlassian.net/browse/DEVX-4911)

## Review

* [Page to review](https://nexmo-developer-pr-3418.herokuapp.com/reports/code-snippets/load-records-sync-id)

### CODE REVIEW

File: _documentation/en/reports/code-snippets/load-records-sync-id.md [hunk 1]
```diff
@@ -17,7 +33,7 @@ Variable | Required | Description
 `ACCOUNT_ID` | Yes | The API key for the target account. Reports generated, or records retrieved, are for this account.
 `REPORT_DIRECTION` | Yes | Either `inbound` or `outbound`
 `REPORT_PRODUCT` | Yes | Specifies the product for which reports and records are obtained. Can be one of `SMS`, `VOICE-CALL`, `WEBSOCKET-CALL`, `VERIFY-API`, `NUMBER-INSIGHT`, `MESSAGES`, `CONVERSATIONS`, or `ASR`.
-`ID` | Yes | The UUID of the message or call to retrieve records for.
+`ID` | Yes | The UUID of the message or call to retrieve record for. It is possible to specify a comma-separated list of UUIDs to retrieve multiple records.
```
Comment (marklewin) [inline _documentation/en/reports/code-snippets/load-records-sync-id.md:36]: Should this be "The UUID of the message or call to retrieve **a** record for"?

**Review by marklewin — CHANGES_REQUESTED** [review]
Querying use of "retrieve record" instead of "retrieve a record", but otherwise looks good!

File: _documentation/en/reports/code-snippets/load-records-sync-id.md (hunk 1 above)
Comment (tbedford) [reply _documentation/en/reports/code-snippets/load-records-sync-id.md]: Well spotted!

**Review by marklewin — APPROVED** [review]
LGTM!

**Pull request merged**